Framework/Algorithms/inc/MantidAlgorithms/ReflectometryBackgroundSubtraction.h

```diff
@@ -0,0 +1,37 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_ALGORITHMS_REFLECTOMETRYBACKGROUNDSUBTRACTION_H_
+#define MANTID_ALGORITHMS_REFLECTOMETRYBACKGROUNDSUBTRACTION_H_
+
+#include "MantidAlgorithms/ReflectometryWorkflowBase2.h"
+
+namespace Mantid {
+namespace Algorithms {
+
+/** ReflectometryBackgroundSubtraction : TODO: DESCRIPTION
+*/
+class DLLExport ReflectometryBackgroundSubtraction
+    : public ReflectometryWorkflowBase2 {
```

Comment:
I don't think we need to inherit from `ReflectometryWorkflowBase2` here (that's aimed more at consolidating code for the higher level algorithms RRO/A and CTW/A, which do a lot of similar things).

Perhaps inherit from `API::DataProcessorAlgorithm` directly or create a new base class if common functions are needed? Although it might be better to copy properties from the child algorithms rather than have a base class with common init functions.